docs/providers.rst

```diff
@@ -1095,6 +1095,8 @@ The following Keycloak settings are available.
 KEYCLOAK_URL:
     The url of your hosted keycloak server, it must end with ``/auth``. For
     example, you can use: ``https://your.keycloak.server/auth``
+    
+    Note: you don't have to use ``/auth`` in the end of URL starting from KeyCloak 17.0.0
```

Comment:
I don't get what this sentence is trying to convey. You don't have to, suggest you still could use it? Or is the intention that you cannot use /auth starting from v17?

Reply:
It doesn't matter anymore, I see that the django-allauth doc was already updated